lookout/style/format/benchmarks/compare_quality_reports.py

```diff
@@ -0,0 +1,94 @@
+import sys
+from typing import TextIO, Union
+
+import pandas
+from tabulate import tabulate
+
+from lookout.style.format.benchmarks.top_repos_quality import FLOAT_PRECISION
+
+
+_column_formats = {
```

Comment:
Idea for the future: use this to reduce the hardcode nightmare which is happening in `top_repos_report.py`

The best solution would be to offload all the tabulation to the Jinja template, but it changes pretty much all the logic, so I am open to compromises.

Reply:
yeah, that is why I decided to create this variable in a module.